src/LightingThread.cpp

```diff
@@ -494,6 +500,35 @@ void cLightingThread::PrepareBlockLight2(void)
 
 
 
+void cLightingThread::PrepareBlockLight3()
```

Comment:
There's was already a `PrepareBlockLight2` but I mainly left it so you can easily distinguish them in profiles.  Should probably just collapse them in the end though.